src/include/pbs_db.h

```diff
@@ -838,18 +508,13 @@ void panic_stop_db(char *txt);
 
 /**
  * @brief
- *	Delete all the server attributes from the database
- *
- * @param[in]	conn - Connection handle
- * @param[in]	obj  - server information
+ *	Translates the error code to an error message
```

Comment:
does panic_stop_db() belong to the pbs_db.h file? In other words is it a DB API? I think it is more of an appliaction/PBS api which then calls DB API's to stop the database, no? If so, it should not be in this file